rubocop-klaxit/cops/postgres_no_between.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module PostgreSQL
+      # Using BETWEEN in PostgreSQL is a bad practice. We a cop to detect
+      # between keyword in string.
+      class NoBetween < Cop
+        MSG = "Do NOT use BETWEEN in PostgreSQL. Please refer: "\
+              "https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_BETWEEN_.28especially_with_timestamps.29"
+
+        # Add offense when detect between in a string
+        def on_str(node)
+          return unless node.value.downcase.match?(/\s{1}\bbetween\b\s{1}/i)
```

Comment:
```suggestion
          return unless node.value.downcase.match?(/\sbetween\s/i)
```

A space is a word boundary, and `{1}` is the default number of characters. Hence the above proposition should be correct!

One other question: why not just `\b`? I had done that in the exemple because the next request is valid:

```sql
select 2 between(1) and 4;
```

Note that `select 2between 1 and 4` is also valid, and is not matched with `\bbetween` unfortunately. Hence I'd be ok with the `\s` proposition, if there is a reason to it!

Reply:
You don't need to downcase the node value, as the reg exp is case insensitive

Reply:
I missed the downcase, good catch!

Reply:
```suggestion
          return unless node.value.match?(/\bbetween\b/i)
```